cppwinrt/component_writers.h

```diff
@@ -988,9 +988,22 @@ namespace winrt::@::implementation
     static void write_generated_static_assert(writer& w)
     {
         auto format = R"(
-// Note: Remove this static_assert after copying these generated source files to your project.
-// This assertion exists to avoid compiling these generated source files directly.
-static_assert(false, "Do not compile generated C++/WinRT source files directly");
+// WARNING: This file is automatically generated from IDL. Do not directly
+// add this file to your project, as any changes you make will be lost.
+//
+// If this is a new file:
+//   1. Copy this file from the 'Generated Files' directory into the
+//      directory where you store your other source files (e.g. the project root).
+//   2. Add the copied file to your project via Project -> Add Existing Item.
+//   3. Delete this comment and the 'static_assert' (below) from the copied file
+//      (do not modify the original file in the 'Generated Files' directory).
```

Comment:
Grammar / punctuation:

```suggestion
//   3. Delete this comment and the 'static_assert' (below) from the copied file.
//      (Do not modify this file.)
```